XCUITests/HomeButtonTests.swift

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+
+class HomeButtonTests: BaseTestCase {
+
+    override func tearDown() {
+        XCUIDevice.shared.orientation = .portrait
+        super.tearDown()
+    }
+
+    func testGoHome() throws {
+        navigator.openURL("https://www.google.com", waitForLoading: true)
```

Comment:
Could we use a local server site? like: https://github.com/mozilla-mobile/firefox-ios/blob/main/XCUITests/TopTabsTest.swift#L12
or https://github.com/mozilla-mobile/firefox-ios/blob/main/XCUITests/TopTabsTest.swift#L26

Reply:
Done.